lib/service/service.go

```diff
@@ -621,8 +621,10 @@ func NewTeleport(cfg *Config) (*TeleportProcess, error) {
 	// create the data directory if it's missing
 	_, err = os.Stat(cfg.DataDir)
 	if os.IsNotExist(err) {
+		cfg.Log.Errorf("Teleport process must have rights to maintain data directory: %v. Run as root or with sudo if not running as process that can maintain that directory. ", cfg.DataDir)
```

Comment:
@r0mant @zmb3 Can you guys think of better error messsage?

Reply:
@zmb3 thank you.  I have corrected the location to be after doing the mkdir and tested again.

Reply:
Okay, we've got the right location now, but I still think we need to work out a better message.

As a security focused company I'm hesitant to say "just run as root if your permissions aren't correct"

Reply:
sorry, resolved before continuing @zmb3 's point.  Should we mention `sudo`, `root` is required just for `ssh` and services.

Reply:
Circling back on this one.  @xinding33 @zmb3 @russjones any suggestions for a error message?